### DISCUSSION TITLE
Where is FastGithub？

### DISCUSSION THREAD

**Discussion by Fgaoxing** [dotnetcore/Home#62] [Q&A]
I can't find it

**Comment by pandaapo** [comment]
I also want to know where is FastGithub now？

**Comment by znsoooo** [comment]
Shh...
[fastgithub_win-x64.zip](https://github.com/dotnetcore/Home/files/13690775/fastgithub_win-x64.zip)

**Reply by pandaapo** [reply]
Thank you for your help. But this security prompt from the OS makes me a bit worried (Google Chrome also prompts for danger when downloading), as it did not appear in a version I previously installed.
![捕获](https://github.com/dotnetcore/Home/assets/35672972/fb37b8f4-4ebf-4e8e-8d67-3c334e12cffa)
